test/tbb/test_task_group.cpp

```diff
@@ -1175,6 +1175,25 @@ TEST_CASE("Task handle empty check"){
     CHECK_MESSAGE(h != nullptr, "delayed task returned by task_group::delayed should not be empty");
 }
 
+//! Test for comparison operations
+//! \brief \ref interface
+TEST_CASE("Task handle comparison/empty checks"){
+    tbb::task_group tg;
+
+    tbb::task_handle h;
+
+    bool empty =  ! static_cast<bool>(h);
```

Comment:
Why do we need an explicit cast here? Just to test it or the code is not compiling without it? I expect it to compile without an explicit cast.

Reply:
Got it! Thanks!